mogc_partnerships/migrations/0003_enrollmentrecord_partner.py

```diff
@@ -0,0 +1,23 @@
+# Generated by Django 3.2.21 on 2024-07-17 19:56
+
+import django.db.models.deletion
+from django.db import migrations, models
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ("mogc_partnerships", "0002_auto_20240703_1419"),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name="enrollmentrecord",
+            name="partner",
+            field=models.ForeignKey(
+                null=True,
```

Comment:
We probably don't _want_ this to be nullable - I think it's possible to backfill during migration, but curious to hear thoughts on doing so vs either leaving it nullable, or merging -> backfill -> new migration to not null as a separate PR.

Reply:
I'm not convinced that we need a new field on enrollment records, but if we decide we do then the `add nullable field` -> `data migration to populate` -> `make non-nullable` migrations can be a part of this PR.